ground-core/pom.xml

```diff
@@ -200,6 +117,13 @@
             <version>1.0.0-1</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/jline/jline -->
```

Comment:
We may not need this. Let's get the alpha out and figure out whether we need this or not.